stylesheets/go-plus.less

```diff
@@ -7,19 +7,19 @@
 .go-plus {
 }
 
-.editor .gutter .go-plus-error {
+.atom-text-editor::shadow .gutter .go-plus-error {
```

Comment:
It should be `atom-text-editor::shadow` (without the dot). See [Upgrading Your UI Theme](https://atom.io/docs/latest/upgrading/upgrading-your-ui-theme).